server/test/account.test.js

```diff
@@ -29,110 +33,203 @@ describe('Account Routes', () => {
-      .expect(401)
-      .end((err, res) => {
-        expect(res.body).to.have.property('status').eql(401);
-        expect(res.body).to.have.property('error').to.eql('User authorization token is expired');
-        expect(res.status).to.equal(401);
-        if (err) return done(err);
-        done();
-      });
+    it('should not create an account with an expired token', (done) => {
+      request(app)
+        .post(`${URL}/accounts`)
+        .send(newAccount)
+        .set('Authorization', `Bearer ${expiredToken}`)
+        .expect(401)
+        .end((err, res) => {
+          expect(res.body).to.have.property('status').eql(401);
+          expect(res.body).to.have.property('error').to.eql('User authorization token is expired');
+          expect(res.status).to.equal(401);
+          if (err) return done(err);
+          done();
+        });
+    });
+
+    it('should not create an account with an invalid token', (done) => {
+      request(app)
+        .post(`${URL}/accounts`)
+        .send(newAccount)
+        .set('Authorization', `Bearer ${fakeToken}`)
+        .expect(401)
+        .end((err, res) => {
+          expect(res.body).to.have.property('status').eql(401);
+          expect(res.body).to.have.property('error').to.eql('Invalid token');
+          expect(res.status).to.equal(401);
+          if (err) return done(err);
+          done();
+        });
+    });
   });
 
-  it('should not create an account with an invalid token', (done) => {
-    request(app)
-      .post(`${URL}/accounts`)
-      .send(newAccount)
-      .set('Authorization', `Bearer ${fakeToken}`)
-      .expect(401)
-      .end((err, res) => {
-        expect(res.body).to.have.property('status').eql(401);
-        expect(res.body).to.have.property('error').to.eql('Invalid token');
-        expect(res.status).to.equal(401);
-        if (err) return done(err);
-        done();
-      });
+  describe('Get Account', () => {
+    let userAccountNumber;
+
+    before((done) => {
+      request(app)
+        .post(`${URL}/accounts`)
+        .send(newAccount)
+        .set('Authorization', `Bearer ${authToken}`)
+        .end((err, res) => {
+          const { accountNumber } = res.body.data;
+          userAccountNumber = accountNumber;
+          if (err) return done(err);
+          done();
+        });
+    });
+
+    it('should get an account with an account number', (done) => {
+      request(app)
+        .get(`${URL}/accounts/${userAccountNumber}`)
+        .set('Authorization', `Bearer ${authToken}`)
+        .expect(200)
+        .end((err, res) => {
+          expect(res.body).to.have.property('status').eql(200);
+          expect(res.body.data).to.have.property('accountNumber').eql(userAccountNumber);
+          expect(res.status).to.equal(200);
+          if (err) return done(err);
+          done();
+        });
+    });
+
+    it('should not get an account with an account number more than 10 digits', (done) => {
```

Comment:
Please also add a test for account numbers less than 10

Reply:
done...